### PR TITLE
feat: enable ledger swaps

### PR DESCRIPTION
**Pull request by bradleySuira**
# :books: Linear Ticket :books:

### CODE REVIEW

File: src/views/Swap/Swap.vue [hunk 1]
```diff
@@ -911,8 +900,8 @@ export default {
       }
       if (
         this.selectedQuote?.receiveFee &&
-        BN(this.selectedQuote.toAmount).lt(
-          BN(this.selectedQuote.receiveFee).times(this.selectedQuote?.maxFeeSlippageMultiplier || 1)
+        BN(this.selectedQuote?.toAmount).lt(
```
Comment (kraikov) [inline src/views/Swap/Swap.vue:903]: `this.selectedQuote` is checked in the first condition
`this.selectedQuote?.receiveFee` so no need to do it after the && statement

**Review by kraikov — CHANGES_REQUESTED** [review]

File: src/views/Swap/Swap.vue (hunk 1 above)
Comment (bradleySuira) [reply src/views/Swap/Swap.vue]: fixed @kraikov

**Review by Jennievon — APPROVED** [review]

**Comment by Jennievon** [comment]
@bradleySuira you got some conflicts with the base branch!

**Review by monokh — APPROVED** [review]

**Review by kraikov — APPROVED** [review]
there is on conflicting file, other than that LGTM

**Pull request merged**